lib/rules/curly.js

```diff
@@ -10,13 +10,24 @@
 
 module.exports = function(context) {
 
-    var multiOnly = (context.options[0] === "multi");
+    var multiOnly = (context.options.indexOf("multi") >= 0);
+    var allowOneLiners = (context.options.indexOf("allow-one-liners") >= 0);
```

Comment:
We typically don't have multiple string options defined in this way. It either needs to replace "multi" or needs to be in an object with properties, such as:

```
curly: [2, "multi", {
    allowSingleLine: true
}
```